wikidata/entity.py

```diff
@@ -188,7 +188,8 @@ def getlist(self, key: 'Entity') -> Sequence[object]:
                          __import__('pprint').pformat(claims))
         decode = self.client.decode_datavalue
         return [decode(snak['datatype'], snak['datavalue'])
-                for snak in (claim['mainsnak'] for claim in claims)]
+                for snak in (claim['mainsnak'] for claim in claims)
+                if snak['snaktype'] == 'value']
```

Comment:
Could you add a test for this case?  See also other tests in *tests/entity_test.py* file.  You could run tests using [tox].

FYI you can make a fixture by fetching `Q8646`:

```bash
curl -o tests/fixtures/entities/Q8646.json \
  https://www.wikidata.org/wiki/Special:EntityData/Q8646.json
```

[tox]: https://tox.readthedocs.io/

Reply:
I added the test case you requested. I verified that it fails before, but succeeds after this change.